README.md

```diff
@@ -35,21 +35,15 @@ Go to [Jhipster Online](https://start.jhipster.tech/generate-azure-application)
 </br> `azd auth login`
 1. Enable Azure Spring Apps feature for AZD
 </br> `azd config set alpha.springapp on`
-1. Prepare environment
-</br> Navigate to the generated project directory and run
+1. Navigate to the generated project directory and run
```

Comment:
shall we add some brief introduction about the function of `azd up` and suggest users to click on the application url generated after running `azd up`?